composer.json

```diff
@@ -49,7 +49,7 @@
         "utopia-php/cache": "0.8.*",
         "utopia-php/cli": "0.14.*",
         "utopia-php/config": "0.2.*",
-        "utopia-php/database": "0.28.*",
+        "utopia-php/database": "dev-increment as 0.28.0",
```

Comment:
Reminder to attach a tag.

Reply:
Waiting database::increment branch to be mearged